book/chapters/dem.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "29dcdd816afc6ba4",
```

Comment:
Please add:

- In the Northern Hemisphere, north-facing slopes (high northness) receive less direct sunlight throughout the year.
- This results in cooler temperatures and more stable microclimates on north-facing slopes.
- South-facing slopes (low or negative northness) receive more direct sunlight, leading to warmer and drier conditions.

Reply:
Eastness

- East-facing slopes (high eastness) receive morning sunlight, which is generally less intense.
- West-facing slopes (low or negative eastness) receive afternoon sunlight, which is typically warmer and more intense.

Reply:
Double check and validate before adding